qt-models/diveplannermodel.h

```diff
@@ -32,6 +32,7 @@ class DivePlannerPointsModel : public QAbstractTableModel {
 	virtual bool setData(const QModelIndex &index, const QVariant &value, int role = Qt::EditRole);
 	virtual Qt::ItemFlags flags(const QModelIndex &index) const;
 	void gaschange(const QModelIndex &index, int newcylinderid);
```

Comment:
hm, `gaschange` basically needs to be camelCase too (`gasChange`). but that;'s from a previous commit.